### ISSUE TITLE
Question about finetuning

### ISSUE THREAD

**Issue by JannikSchneider12** [BEAM-Labs/ContraNovo#1]
Hello everyone,

I wanted to use ContraNovo for my mass spec data and want to finetune it based on my labeled spectra but have some questions. Am I right that I just need to set the "train_by_scratch" parameter to false and that my mgf files include the "params" "seq" parameter? 
And I am not sure how to use a training and validation set, since I have not seen an input parameter for this.

Thanks for your time and help
